build_omaha.py

```diff
@@ -99,6 +113,8 @@ def ParseArgs():
                       nargs=1)
   parser.add_argument('--tag_ap',
                       nargs=1)
+  # parser.add_argument('--tag_admin',
```

Comment:
nit: This would not be used because you introduced `TAG_ADMIN` env.

Reply:
Removed, I'll merge as soon as you approve again.